src/lib/select/select.spec.ts

```diff
@@ -613,6 +613,34 @@ describe('MatSelect', () => {
         expect(() => fixture.componentInstance.select.open()).not.toThrow();
       });
 
+      it('should set the width of the overlay based on the trigger and resize', async(() => {
+        trigger.style.width = '200px';
+        fixture.detectChanges();
+        fixture.whenStable().then(() => {
```

Comment:
In general we're moving away from `async` tests, because they tend to be slower and flakier. Can you try to turn this into a `fakeAsync` one? All you have to do is replace the `fixture.whenStable` callbacks with a `flush` call.

Reply:
Cool, will do that